TODO

```diff
@@ -1,8 +1,5 @@
-
 - Remove custom .desktop file:
   https://github.com/NationalSecurityAgency/ghidra/pull/1556
-- 2x support?
-  https://github.com/bkerler/ghidra_installer/blob/master/install-ghidra.sh#L83-L88
```

Comment:
What allows you to remove the TODO item? Is it fixed by the previous commit? If so, you might as well merge those 2 commits, so it's clear when the problem was fixed.

Reply:
Yeah, this should be fixed by commit 5ef862d1a0c79d27dbb6c7099e8e4bf3a5f58ef3.

Reply:
Merged